react/features/app/middlewares.any.js

```diff
@@ -19,7 +19,7 @@ import '../base/sounds/middleware';
 import '../base/testing/middleware';
 import '../base/tracks/middleware';
 import '../base/user-interaction/middleware';
-import '../breakout-rooms/middleware';
+import '../participants-pane/components/breakout-rooms/middleware';
```

Comment:
Don't move the middleware there, leave it and the reducer in features/breakrout-rooms.